pages/en/lb3/Using-current-context.md

```diff
@@ -70,6 +70,33 @@ remoting metadata and set the `http` property to the special string value
 }
 ```
 
+For example:
+```js
+  MyObject.remoteMethod(
+    'myMethod',
+    {
+      http: {path: '/myMethod', verb: 'get'},
+      accepts: [
+        {arg: 'someArgument', type: 'string'},
+        {
+          arg: 'accessToken',
+          type: 'object',
+          http: function(ctx) {
+            return ctx.req.accessToken;
+          }
+        }
```

Comment:
Admittedly, I am a newbie to stuff so just trying to wrap my head around it. I've been reading about various approaches to context and using the access token, and it appears that many people are also in the same boat.

With regards to the access token and making authenticated requests, I eventually landed on putting the access token in the request header as the Authorization parameter rather than in the request body as the "access_token" parameter. Various articles seemed to say this approach is a bit safer.

With the recommended options approach to current context, why do I need to send options in the request when I'm already sending the access token in the request header?